### PR TITLE
Lesson6 solution

### PR DESCRIPTION
**Pull request by Ognessa**

### CODE REVIEW

File: app/src/main/java/com/onix/internship/ui/splash/SplashFragment.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@ class SplashFragment : BaseFragment<SplashFragmentBinding>(R.layout.splash_fragm
     }
 
     private fun showLogInScreen() {
```
Comment (shadwork) [inline app/src/main/java/com/onix/internship/ui/splash/SplashFragment.kt:19]: rename please

File: app/src/main/java/com/onix/internship/services/BackgroundSoundService.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        return null
+    }
+
+    override fun onCreate() {
+        super.onCreate()
+        val afd = applicationContext.assets.openFd("illurock.opus")
+        player = MediaPlayer()
+        player.setDataSource(afd.fileDescriptor)
+        player.isLooping = true // Set looping
+        player.setVolume(100f, 100f)
```
[the remark this answers is not in the document]
Comment (Ognessa) [reply app/src/main/java/com/onix/internship/services/BackgroundSoundService.kt]: Done

File: app/src/main/java/com/onix/internship/ui/pause/PauseFragment.kt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    private fun continueGame(){
+        viewModel.stepBack()
+        val action = PauseFragmentDirections.actionPauseFragmentToGameFragment()
+        findNavController().navigate(action)
+    }
+
+    private fun restartGame(){
+        viewModel.clearData()
+        val action = PauseFragmentDirections.actionPauseFragmentToGameFragment()
```
[the remark this answers is not in the document]
Comment (Ognessa) [reply app/src/main/java/com/onix/internship/ui/pause/PauseFragment.kt]: done

File: app/src/main/java/com/onix/internship/ui/splash/SplashFragment.kt (hunk 1 above)
Comment (Ognessa) [reply app/src/main/java/com/onix/internship/ui/splash/SplashFragment.kt]: done

File: app/src/main/java/com/onix/internship/ui/game/GameFragment.kt [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+                binding.ivGameBackground.setImageDrawable(drawable)
+            }
+            else{
+                binding.ivGameBackground.setImageDrawable(null)
+            }
+        }
+        viewModel.currentCharacterImage.observe(this){
+            if(!it.isNullOrEmpty()){
+                val drawable = requireContext().resources.getDrawable(requireContext().resources
+                    .getIdentifier(it, "drawable", requireContext().packageName), null)
```
[the remark this answers is not in the document]
Comment (shadwork) [reply app/src/main/java/com/onix/internship/ui/game/GameFragment.kt]: yep

**Pull request merged**